### DISCUSSION TITLE
Publication of nmaglev

### DISCUSSION THREAD

**Discussion by tak30** [nomasystems/nmaglev#9] [Announcements]
Please mark the check once you think it is ok for publication.
- [ ] @AidaAP 
- [x] @Alberdi 
- [ ] @DavidGonzalezPineiro 
- [ ] @LoisSotoLopez 
- [x] @miguelrubinos / @mrrubinos 
- [x] @mpquique 
- [x] @tak30 
- [x] @villaverde82 
- [ ] @josecriane 
- [x] @javiergarea 
- [ ] @bobergj

**Comment by tak30** [comment]
Removed common test duplicated code, using now nct_util: https://github.com/nomasystems/nmaglev/commit/6385ce0c5706e4ae25c96950249784270898e08e

**Comment by bobergj** [comment]
* it's not really implementing the whole maglev paper, but a small part of it (the balancing algorithm at the core of it). That's fine but it's referencing the paper in the README without explaining exactly what it does.
* public api should have typespecs, last time I did Erlang at least that was expected in the general Erlang community
* tests could be better by adding some tests that approach the public API from a use-case point of view
* test badge in README isn't linking to test results

**Reply by tak30** [reply]
I agree, I'll create an issue to approach the proposed changes. thanks!
